### ISSUE TITLE
Unauthorized Extension

### ISSUE THREAD

**Issue by frutiger1** [Fugiman/google-meet-grid-view#303]
After installing GridView 1.39 there is a large warning message in the extension that it is not an authorized extension and to install the official version which links to 1.37. Is there any way to remove this message from the extension installed from the chrome webstore, or is there some other way I would need to deploy?

**Comment by Eagle-Man** [comment]
No.  Since the original creator is MIA, no one else can make modifications to this extension (to update or remove), therefore any new version posted will be seen as unofficial.

**Comment by FunkyBrewster** [comment]
Maybe there should be a time limit on abandoned code so that it becomes transferable to public domain if the author is deemed to be uncontactable.
